lib/widgets/fields/padding_field/padding_text_fields.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+
+class PaddingTextField extends StatelessWidget {
+  final double fontSize, fieldWidth, fieldHeight;
```

Comment:
Do not write variables in a single line, always spread them in multiple lines like following
```
final double fontSize;
final double fieldWidth;
final double fieldHeight;
```

Also, I don't think providing field width and height is required here. This should be determined by the font size itself.